### ISSUE TITLE
import from other apps

### ISSUE THREAD

**Issue by corobin** [freeotp/freeotp-android#302]
is it possible to import token databases from other apps like andotp/aegis? since those apps do in fact have functionality to export backups whether we like it or not, perhaps freeotp can support importing from them?

**Comment by ajdelgado** [comment]
No, the format is different. I'm working on a python script to import from andotp (and possibly also from Gnome Authenticator) as soon as I can find out some details I asked in https://github.com/freeotp/freeotp-android/issues/368

**Comment by jimmysjolund** [comment]
Is this answered perhaps?
